apps/frontend/src/utils/math.ts

```diff
@@ -0,0 +1,29 @@
+import { isBigNumberish } from '@ethersproject/bignumber/lib/bignumber';
+
+import { BigNumberish, BigNumber } from 'ethers';
+import { formatUnits, parseUnits } from 'ethers/lib/utils';
+
+// helper function to convert any type of ethers value to wei.
+export const toWei = (value: BigNumberish): BigNumber => {
+  if (isBigNumberish(value)) {
+    return BigNumber.from(value).mul(BigNumber.from(10).pow(18));
```

Comment:
I think that this depends on a number of decimals, right? I know that 18 is the most common but what if we have a token with a different amount of decimals?

Reply:
good point, will add argument for overriding decimals points.